oauth.go

```diff
@@ -1,14 +1,14 @@
 package goshopify
```

Comment:
I would revert this file.

Reply:
Somehow I wasn't able to. I did `git checkout origin/master oauth.go` and confirmed it got reset. When I did git push again this file shows up.

Reply:
It actually might have been a git hook from devtools.